docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -0,0 +1,454 @@
+
+### Deserialization of Object properties
+
+When `Newtonsoft.Json` deserializes to `Object` properties in POCOs or in dictionaries of type `Dictionary<string, Object>`, it:
+
+* "Guesses" the type of "primitive" values in the JSON and stores `string`/`long`/`double`/`boolean`/`DateTime` as a boxed object.
+* Stores "complex" values as `JObject`.
+* Stores `null` when the payload has "null".
+
+`System.Text.Json` stores a boxed `JsonElement` for "primitive" and "complex" values within the System.Object property or dictionary value. It stores `null` when the payload has "null" in it.
+
+To implement type inference for `Object` properties, create a converter like the example in [How to write custom converters](system-text-json-converters-how-to.md#deserialize-inferred-types-to-object-properties).
+
+### Maximum depth
+
+`Newtonsoft.Json` doesn't have a maximum depth limit. For `System.Text.Json` there's a default limit  of 64, and it's configurable by setting (xref:System.Text.Json.JsonSerializerOptions.MaxDepth?displayProperty=nameWithType).
+
+## Scenarios using JsonSerializer that require workarounds
+
+The following scenarios aren't supported by built-in functionality, but sample code is provided for workarounds.
+
+### Specify date format
+
+`Newtonsoft.Json` provides a `DateTimeZoneHandling` option that lets you specify the serialization format used for `DateTime` and `DateTimeOffset` properties. In `System.Text.Json`, the only format that has built-in support is ISO 8601-1:2019. To use any other format, create a custom converter. For more information, see [DateTime and DateTimeOffset support in System.Text.Json](../datetime/system-text-json-support.md).
+
+### Quoted numbers
+
+`Newtonsoft.Json` can serialize or deserialize numbers in quotes. For example, it can accept: `{ "DegreesCelsius":"23" }` instead of `{ "DegreesCelsius":23 }`. To enable that behavior in `System.Text.Json`, implement a custom converter like the following example, which uses quotes for properties defined as `long`.
+
+[!code-csharp[](~/samples/snippets/core/system-text-json/csharp/LongToStringConverter.cs)]
+
+[Register this custom converter](system-text-json-converters-how-to.md#register-a-custom-converter) by using an attribute on individual `long` properties or by adding the converter to the `Converters` collection.
```

Comment:
Link `Converters` to https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions.converters?view=netcore-3.1